actions/maven_build_and_tag/action.yml

```diff
@@ -53,7 +53,7 @@ runs:
       # Option -U (or --update-snapshots)
       #   Forces a check for missing releases and updated snapshots on remote repositories
       #   Required because using caching of Maven dependencies
-      run: mvn -B -U --file ${{ inputs.pom-xml-file }} ${{ inputs.maven_goals }} ${{ inputs.maven_other_options }}
+      run: mvn -B -U --file ${{ inputs.pom-xml-file }} ${{ inputs.maven_goals }} ${{ inputs.maven_other_options }} | tee mvn_results.txt
```

Comment:
File mvn_results.txt here is used to communicate between this action and send email action, but it is implicit: it assumes that both actions write and read to same place, and path in not really specified since relative to current execution context
I would:
* add parameter to this action with path where to write file (path using on $github_workspace/...)
* write to this file here
* pass also variable to send_email_action, for example a parameter named `body_file` , and have action read from it
This way actions write and read where they are told, they don't have to make asummptions about other pieces